Added support for ubuntu linux desktop

raks437 opened the pull request:
Created test cases for latest firefox and chromium browser running on Ubuntu linux.
Added support to identify Ubuntu linux os.
updated the readme to reflect support is added for Ubuntu.
Removed a delimeter after a for block (I have no idea why it was there but my editor keep throwing warning for that).

wilddeer commented:
Thanks for the pull request!

Two things to take care about though:

- Please, remove all dist/* files from your PR, I'll build them myself (I should've mentioned this in readme, but I didn't expect any PRs, really)

- You shouldn't use version search (`$version: {search: 'Ubuntu'}`) since there's no Ubuntu version information in both UA strings

raks437 commented:
got it..will take care of these points from the next PRs..I am hoping to add more..

raks437 commented:
Implemented the last advised change and added Fedora Linux Desktop OS.

The changes to `dist/` are added already because I've added those file in this PR already. You could always do `grunt build` to overwrite and rebuild them again. Let me know what suits you ?
